wandb/sdk/launch/agent/agent.py

```diff
@@ -105,30 +108,29 @@ def _max_from_config(
     return max_from_config
 
 
-def _is_scheduler_job(run_spec: Dict[str, Any]) -> bool:
-    """Determine whether a job/runSpec is a sweep scheduler."""
-    if not run_spec:
-        _logger.debug("Recieved runSpec in _is_scheduler_job that was empty")
+class InternalAgentLogger:
+    def __init__(self, verbosity=0):
+        self.verbosity = verbosity
```

Comment:
minor suggestion is to maybe add internal variable:
`self._to_terminal = verbosity >=2` (or whatever name you want, this way you don't need to update it everywhere, just in one place)
what do you think?